src/foam/u2/crunch/lab/CrunchLab.js

```diff
@@ -45,41 +47,56 @@ foam.CLASS({
 
   properties: [
     {
-      name: 'rootCapability',
+      name: 'crunchUser',
       class: 'Reference',
-      of: 'foam.nanos.crunch.Capability',
+      of: 'foam.nanos.auth.User',
       view: function(_, X) {
         return {
           class: 'foam.u2.view.RichChoiceView',
           search: true,
           sections: [
             {
-              heading: 'Capabilities',
-              dao: X.capabilityDAO
+              heading: 'Users',
+              dao: X.userDAO
             }
           ]
         };
       },
-      // postSet: function(_, n) {
-      //   this.memento = n;
-      // }
     },
     {
-      name: 'crunchUser',
+      class: 'foam.dao.DAOProperty',
+      name: 'filteredCapabilityDAO',
+      expression: function(crunchUser) {
+        if ( crunchUser == 0 ) return this.capabilityDAO;
+        return this.PromisedDAO.create({
+          of: 'foam.nanos.crunch.Capability',
+          promise: this.userCapabilityJunctionDAO.where(this.EQ(this.UserCapabilityJunction.SOURCE_ID, crunchUser))
+            .select(this.MAP(this.UserCapabilityJunction.TARGET_ID))
+            .then((sink) => {
+              let capabilities = sink.delegate.array ? sink.delegate.array : [];
```

Comment:
When does this case happen?

Reply:
I'm not completely sure, but I've encounter cases.

Reply:
Not too sure why that case happens but I remember encountering it when similar logic was applied on the BankAccount currencies.